### ISSUE TITLE
The RPM package is not noarch 

### ISSUE THREAD

**Issue by Fuchange** [api7/build-tools#3]
I note that [apisix-2.1](https://github.com/apache/apisix/releases/tag/2.1) provided a noarch rpm package, 

But it contains some binaries files built on X86_64 and caused the apifix command cant find the file at ARM system.

Like [apache/apisix#issue-691844591](https://github.com/apache/apisix/issues/2161#issue-691844591).

```bash 
# file ./local/apisix/deps/lib/lua/5.1/lfs.so
./local/apisix/deps/lib/lua/5.1/lfs.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=5de2ad9d73ddbdaa91151ddfee3eef545a010902, not stripped
``` 

Would you like to add some options to fpm in run.sh to buind different arch packages? 
Or I can provided an spec file to build packages by rpmbuild command ?

**Comment by membphis** [comment]
Which cloud platform do you use for ARM compilation?

**Comment by Fuchange** [comment]
@membphis  I build it on my local ARM server. 

If I build it locally, it's OK.

**Comment by imjoey** [comment]
> Which cloud platform do you use for ARM compilation?

Hi @membphis , we are using a physical server with Kunpeng 920.

**Comment by membphis** [comment]
@idbeta do you have time to look at this issue?

**Comment by idbeta** [comment]
> @idbeta do you have time to look at this issue?

Yes, but I don’t have this kind of environment now, @imjoey can you provide an environment to test it. My qq number is `83434649`

**Comment by idbeta** [comment]
> I note that [apisix-2.1](https://github.com/apache/apisix/releases/tag/2.1) provided a noarch rpm package,
> 
> But it contains some binaries files built on X86_64 and caused the apifix command cant find the file at ARM system.
> 
> Like [apache/apisix#issue-691844591](https://github.com/apache/apisix/issues/2161#issue-691844591).
> 
> ```shell
> # file ./local/apisix/deps/lib/lua/5.1/lfs.so
> ./local/apisix/deps/lib/lua/5.1/lfs.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=5de2ad9d73ddbdaa91151ddfee3eef545a010902, not stripped
> ```
> 
> Would you like to add some options to fpm in run.sh to buind different arch packages?
> Or I can provided an spec file to build packages by rpmbuild command ?

I got your point, the RPM package provided now is indeed not noarch, the name needs to be changed. @moonming

**Comment by imjoey** [comment]
> > I note that [apisix-2.1](https://github.com/apache/apisix/releases/tag/2.1) provided a noarch rpm package,
> > But it contains some binaries files built on X86_64 and caused the apifix command cant find the file at ARM system.
> > Like [apache/apisix#issue-691844591](https://github.com/apache/apisix/issues/2161#issue-691844591).
> > ```shell
> > # file ./local/apisix/deps/lib/lua/5.1/lfs.so
> > ./local/apisix/deps/lib/lua/5.1/lfs.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=5de2ad9d73ddbdaa91151ddfee3eef545a010902, not stripped
> > ```
> > 
> > 
> > Would you like to add some options to fpm in run.sh to buind different arch packages?
> > Or I can provided an spec file to build packages by rpmbuild command ?
> 
> I got your point, the RPM package provided now is indeed not noarch, the name needs to be changed. @moonming

@idbeta yep, exactly. We are supposed to make independent rpms for each cpu arch platforms, `x86_64` and `aarch64` for now.

I guess @Fuchange and I could help to commit a pr to fix this issue.

> @idbeta  We are working to bring OpenResty as well as APISIX to openEuler community, please refer to https://gitee.com/openeuler/community/pulls/1240 for details. Welcome to join us.

**Comment by idbeta** [comment]
> @idbeta yep, exactly. We are supposed to make independent rpms for each cpu arch platforms, `x86_64` and `aarch64` for now.
> 
> I guess @Fuchange and I could help to commit a pr to fix this issue.
> 
> > @idbeta  We are working to bring OpenResty as well as APISIX to openEuler community, please refer to https://gitee.com/openeuler/community/pulls/1240 for details. Welcome to join us.

Thanks, I will go to your community to learn.

**Comment by Fuchange** [comment]
@membphis  @idbeta  

Hello everyone ,I merged a pr(#4) to fix it ,pls have a look~

**Comment by membphis** [comment]
nice, many thx @Fuchange
